UNI_EXT/SoulstormHandlers.h

```diff
@@ -0,0 +1,7 @@
+// Based on slashdiablo-maphack implementation
+// https://github.com/planqi/slashdiablo-maphack
+
+#pragma once
+#include <Windows.h>
+
+DWORD WINAPI MainThread(VOID* lpvoid);
```

Comment:
Same deal. Can this be removed or is this still necessary?

Reply:
You're right. It was leftover code. I removed it and it seems library compiles and loads.